README.md

```diff
@@ -1,86 +1,228 @@
- 
-## Special Thanks
+```js
+const emoji = require('node-emoji')
 
-... to Anand Chowdhary (@AnandChowdhary) and his company Pabio (https://github.com/pabio) for sponsoring this project via [GitHub Sponsors](https://github.com/sponsors/omnidan)!
+console.log(emoji.get('unicorn'))
+// '🦄'
+```
+
+### emoji.has(emoji)
+
+Check if this library supports a specific emoji.
+
+Parameters:
+
+1. **`emoji`** (`string`): The emoji to check.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.has('🦄'))
+// true
+```
+
+### emoji.random()
+
+Get a random emoji.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.random())
+// { name: 'unicorn', emoji: '🦄' }
+```
+
+### emoji.replace(input, replacement)
+
+Replace the emojis in a string.
+
+Parameters:
+
+- **`input`** (`string`): The input string.
+- **`replacement`** (`string | (emoji: string, index: number, string: InputValueType) => string`): The character to replace the emoji with. Can be either a string or a callback that returns a string.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.replace('The 🦄 is a fictitious animal.', 'unicorn'))
+// 'The unicorn is a fictitious animal.'
+```
+
+### emoji.search(keyword)
+
+Search for emojis containing the provided name in their name.
+
+Parameters:
+
+1. **`keyword`** (`string`): The keyword to search for.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.search('honey'))
+// [ { name: 'honeybee', emoji: '🐝' }, { name: 'honey_pot', emoji: '🍯' } ]
+```
+
+### emoji.strip(input, options?)
+
+Remove all of the emojis from a string.
+
+Parameters:
+
+1. **`input`** (`string`): The input string to strip the emojis from.
+1. **`options`** _(optional)_:
+
+   - **`preserveSpaces`** (`boolean`): Whether to keep the extra space after a stripped emoji.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.strip('🦄 The unicorn is a fictitious animal.'))
+// emoji.strip('The unicorn is a fictitious animal.')
+
+console.log(
+  emoji.strip('🦄 The unicorn is a fictitious animal.', {
+    preserveSpaces: true,
+  })
+)
+// emoji.strip(' The unicorn is a fictitious animal.')
+```
+
+### emoji.unemojify(input)
+
+Convert all emojis in a string to their markdown-encoded counterparts.
+
+Parameters:
+
+1. **`input`** (`string`): The input string containing the emojis.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.unemojify('The 🦄 is a fictitious animal.'))
+// emoji.strip('The :unicorn: is a fictitious animal.')
+```
+
+### emoji.which(emoji)
+
+Get an emoji name from an emoji.
+
+Parameters:
+
+1. **`emoji`** (`string`): The emoji to get the name of.
+
+```js
+const emoji = require('node-emoji')
+
+console.log(emoji.which('🦄'))
+// 'unicorn'
+```
 
 ## License
+
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fomnidan%2Fnode-emoji.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2Fomnidan%2Fnode-emoji?ref=badge_large)
```

Comment:
Given that this is just a badge, it could be moved into the header

Reply:
This is existing code 🙂

Reply:
It's pretty huge - so unless @omnidan wants to trim down I'm inclined to leave it where it is.